doc/mgr/orchestrator.rst

```diff
@@ -757,7 +757,7 @@ This is an overview of the current implementation status of the orchestrators.
  device ls                           ✔      ✔
  iscsi add                           ⚪     ✔
  mds add                             ✔      ✔
- nfs add                             ✔      ✔
+ nfs add                             ⚪      ✔
```

Comment:
Isn't this PR fixing the nfs add scenario? Seems like it should still be checked?

Reply:
Ok, I see now the `apply` is preferred